src/app/screens/Home/layout.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Book} from "../../components/Book/index.js"
+import {Selector} from "./components/Selector/index.js"
+import {Searcher} from "./components/Searcher/index.js"
```

Comment:
if you export the searcher as default in Searcher/index.js, you can import it like this

`import Searcher from './components/Searcher';` (by default, javascript checks in index.js)

same for all imports